template-contacts-new.php

```diff
@@ -24,25 +24,25 @@
                     <button class="help-button" data-section="contact-name-help-text">
                         <img class="help-icon" src="<?php echo esc_html( get_template_directory_uri() . '/dt-assets/images/help.svg' ) ?>"/>
                     </button>
-                    <input name="title" type="text" placeholder="<?php esc_html_e( "Name", "disciple_tools" ); ?>" required dir="auto" aria-describedby="name-help-text">
+                    <input name="title" type="text" placeholder="<?php echo esc_html_x( "Name", 'input field placeholder', 'disciple_tools' ); ?>" required dir="auto" aria-describedby="name-help-text">
                 </label>
-                <p class="help-text" id="name-help-text"><?php esc_html_e( "This is required", "disciple_tools" ); ?></p>
+                <p class="help-text" id="name-help-text"><?php echo esc_html_x( "This is required", "disciple_tools" ); ?></p>
```

Comment:
missing context

Reply:
`<?php esc_html_e( "This is required", "disciple_tools" ); ?>`
Interesting. My code has this and not what you quoted. Not sure why. Anyway, I think without context is better as it isn't too hard to translate.